src/components/ChannelList/ChannelList.tsx

```diff
@@ -308,6 +324,11 @@ export const ChannelList = <
     setChannels,
   });
 
+  useChannelVisible({
```

Comment:
this should be after useChannelUpdated. Just to adhere to our alphabetical order